Public/Src/FrontEnd/MsBuild/PipConstructor.cs

```diff
@@ -330,6 +330,16 @@ public bool TrySchedulePipForFile(ProjectWithPredictions project, QualifierId qu
                     continue;
                 }
 
+                // Nuget restore generates nuget.g.props files that have absolute paths embedded. This blocks shared cache from working when
+                // the project file (and therefore the corresponding .pkgrefgen folder) is placed on machine-dependent folders
+                // Even though untracking this file is not completely safe from a caching perspective, in practice this should be harmless

```

Comment:
Harmless because any cache invalidation changes on the `nuget.g.props` would also imply changes to `project.assets.json`?

Reply:
Exactly